python/rikai/types/geometry.py

```diff
@@ -281,6 +281,36 @@ def _area(arr: np.ndarray) -> np.array:
             0, arr[:, 3] - arr[:, 1]
         )
 
+    @staticmethod
+    def iou_matrix(box_list1: Union[Sequence[Box2d], np.ndarray],
```

Comment:
lets rename them as `boxes1, boxes2` ? they are not necessary a `list` ;)

Reply:
Shall we just renames this as `ious`, and allow the function takes `Union[Sequence[Box2d], np.ndarray, Box2d]`?